File: C++/Problems/BASICS/05_PrimeNumberOrNot.cpp
#include<iostream>
using namespace std;
int main(){
    int n,count;
    cout<<"Enter a number to check it is prime or not : ";
    cin>>n;
    count = 0;
    for (int i=1; i<=n ;i++){
        if(n % i == 0){
            count = count + 1;
        }
    }
if(count == 2){
    cout<<"This is a Prime Number";
}
else{
    cout<<"This is not a Prime Number";
}

return 0;
}

File: Python/Problems/BASICS/05_PrimeNumberOrNot.py
n= int(input("Enter a number to check it is prime or not : "))
count = 0
for i in range(1,n+1):
    if (n%i==0):
        count = count + 1
if (count <= 2):
    print(n," is a Prime number")
else:
    print(n," is not a Prime number")
